### DISCUSSION TITLE
Unable to change rofi theme to my custom theme

### DISCUSSION THREAD

**Discussion by digvijayizing** [JaKooLit/Hyprland-Dots#315] [Q&A]
I want to know how to change the rofi theme for example i want to apply a theme i found in another rice , even when i pasted the cli file and it did showed in the rofi theme selector it didnt worked i dont know if its a bug or not

**Comment by JaKooLit** [comment]
move this to discussion because this is not a bug


if you want to change to your own theme

change all the config.rasi in

~/.config/rofi/

**Reply by digvijayizing** [reply]
all ? shouldn't copying my desired will work ? and sorry for posting this here i'll keep this in mind to write such things in discussions

**Comment by JaKooLit** [comment]
there is a lot of how to out there

this is one of the result
https://github.com/adi1090x/rofi

just remember that rofi was used a menu for almost everything... so youll have to adjust it as required


and all rofi related themes and configs are in the directory I mentioned above

unfortunately, I cannot write a how to or whatever, but as Ive said, all those rofi related stuff are collected in that directory
